extensions/miro/src/remove-member.tsx

```diff
@@ -0,0 +1,31 @@
+import { Action, ActionPanel, Form, showToast, Toast, useNavigation } from "@raycast/api";
+import ListMembers from "./list-member";
+import * as miro from "./oauth/miro";
+
+// remove member react component
+export default function RemoveMember({ id, memberId }: { id: string; memberId: string }) {
```

Comment:
instead of using a component for that, you can use an `Alert`: https://developers.raycast.com/api-reference/feedback/alert

especially because in the action here, you are pushing a new view so if the user presses `esc` after, they will come back to this screen

Reply:
Ah didn't know this API, will do, that's def better.